modules/core/src/main/java/org/apache/ignite/internal/jdbc/thin/JdbcThinConnection.java

```diff
@@ -1433,6 +1501,16 @@ void readResponses() {
                             break;
                         }
                     }
+                    else if (resp.response() instanceof JdbcBinaryTypeGetResult)
+                        metaHnd.handleResult((JdbcBinaryTypeGetResult)resp.response());
+
+                    else if (resp.response() instanceof JdbcBinaryTypeNameGetResult)
+                        marshCtx.handleResult((JdbcBinaryTypeNameGetResult)resp.response());
+
+                    else if (resp.response() instanceof JdbcUpdateBinarySchemaResult) {
+                        if (!marshCtx.handleResult((JdbcUpdateBinarySchemaResult)resp.response()))
+                            metaHnd.handleResult((JdbcUpdateBinarySchemaResult)resp.response());
```

Comment:
Do we need to handle somehow a case when neither `marshCtx` nor `metaHnd` handled result returning _true_?

Reply:
There is only 2 way to get this situation:
1) Server send this message by mistake (something very strange happened);
2) Message with the same request ID already handled (possible the message was resended? ).

These both seems very rare (or impossible) cases. But I suppose we can add some logging. What do you think?

Reply:
Logging sounds as good option to me.

Reply:
done.